### PR TITLE
ssp_206_config

### PR DESCRIPTION
**Pull request by ioigoume**

### CODE REVIEW

File: roles/php/vars/Debian-bookworm.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Only when php_alt_repo is set to true
+php_apt_repositories:
+  - "deb https://packages.sury.org/php/ bookworm main"
+
+php_apt_key: "https://packages.sury.org/php/apt.gpg"
+
+php_apt_packages:
+  - apt-transport-https
+  - ca-certificates
```
Comment (NicolasLiampotis) [inline roles/php/vars/Debian-bookworm.yml:18]: This change should be in a separate PR which adds support for Debian bookworm to the php role

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Please see comments and use `2.0` (i.e. {Major-version}.{Minor-version}) in the filenames for config templates

File: roles/php/vars/Debian-bookworm.yml (hunk 1 above)
Comment (ioigoume) [reply roles/php/vars/Debian-bookworm.yml:18]: ok

File: roles/ssp/templates/config/authsources-2.0.6.php.j2
Comment (NicolasLiampotis) [inline roles/ssp/templates/config/authsources-2.0.6.php.j2:1]: @ioigoume  Could you rename this to `authsources-2.0.php.j2` and add `authsources-2.1.php.j2`?

Comment (ioigoume) [reply roles/ssp/templates/config/authsources-2.0.6.php.j2:1]: done

File: roles/ssp/defaults/main.yml
Comment (NicolasLiampotis) [inline roles/ssp/defaults/main.yml:1]: These changes are not related to the config templates for v2.x and can be removed from this PR

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
With the recent commit [a9dbe03](https://github.com/rciam/rciam-deploy/commit/a9dbe03bcf5462198cebd705f71ab685623b8db2) there's no longer a need for duplicate templates for both versions 2.0 and 2.1.

Please adjust this PR by including only a single version of each configuration template for SSPv2, named <config-name>-2.php.j2, if there are no differences between versions 2.0 and 2.1. Additionally, if any templates are identical to those used for SSPv1, you can include them as <config-name>.php.j2, omitting the version number from their filename.

**Review by NicolasLiampotis — APPROVED** [review]
LGTM
